### DISCUSSION TITLE
Usage report of free Codespaces usage

### DISCUSSION THREAD

**Discussion by aschilling** [community/community#38260] [Codespaces]
Hi everybody,

I am currently using Github Codespaces as a teacher with the educational allowance. It would help me a lot to see the current usage of Codespaces within my free allowance, so that I can judge whether I can expand the usage to another class. Is there any way to get the current usage within the free allowance? If I follow the official guide for looking up the Codespace usage for my classroom, all usage figures are zero although I used Codespaces this week in a classroom setting.

Best regards
Andy

**Comment by jfullstackdev** [comment]
this is I think the limit 

https://docs.github.com/en/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide/using-github-codespaces-with-github-classroom#about-the-codespaces-education-benefit-for-verified-teachers

from that, you can estimate your usage

**Reply by aschilling** [reply]
Hi @jdevstatic thanks for the reply. I know the free allowance, but what I am missing is a way to monitor the current usage so that I can judge whether there is enough allowance to use Codespaces in another class. Sorry for the confusion.

**Comment by fara-nak** [comment]
Hi @aschilling , unfortunately we don't have this feature yet. However, the Codespaces Education benefit is currently in public beta and teacher  will not be charged if they exceed the free allowance.

**Reply by aschilling** [reply]
Hi @fara-nak thank you very much for your reply. One thing which is unclear to me is the following: If the educational alowance is used, does codespaces automatically use the [free personal allowance](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-github-codespaces#codespaces-pricing) of the student?
